doc/source/management/data_at_rest_encryption.rst

```diff
@@ -79,15 +89,36 @@ To change the default value use the following syntax:
 
 Here, **NEW_ID** is an unsigned 32-bit integer.
 
+
 .. _data-at-rest-encryption.innodb-system-tablespace:
 
+Encryption threads
+========================================================================
```

Comment:
I don't think this feature has been fully implemented (and/or released yet)

Reply:
OK. Not sure what to do here. I can rewrite the section to remove "encryption threads"?

Reply:
Yes, encryption threads are not GA and really not even functionally experimental

Reply:
Rewritten to remove any reference.